yt_idv/scene_components/blocks.py

```diff
@@ -7,11 +7,12 @@
 from yt_idv.gui_support import add_popup_help
 from yt_idv.opengl_support import TransferFunctionTexture
 from yt_idv.scene_components.base_component import SceneComponent
+from yt_idv.scene_components.isolayers import Isolayers
 from yt_idv.scene_data.block_collection import BlockCollection
 from yt_idv.shader_objects import component_shaders
 
 
-class BlockRendering(SceneComponent):
+class BlockRendering(SceneComponent, Isolayers):
```

Comment:
@matthewturk any opinio on using a multi-inheritence structure here? was the most convenient way i saw to add the isolayer functionalty to the block rendering while allowing future re-use with octree blocks.

Reply:
(which is why we had initially added it to the base component I think)

Reply:
hmm, good point.  I guess I'm OK with it, but I also wonder if this should just be either a different shader (which I guess would open up multiple issues with having new uniforms that are conditionally modified) or a different component referencing the same data object.

Reply:
I think a different component is actually a great idea. And we could have a "add isolayer" button similar to the button for adding a grid outline. And it would allow viewing both isolayers and a volume rendering at once (though I'm not sure this would work right away... Offhand I don't remember how the different components are blended, might be some depth buffer issues...).